public/components/search-field/date-search.pug

```diff
@@ -4,48 +4,54 @@
     input.input-sm.form-control.margin-top(
         type="text", 
         ng-model="dateModel.date.query.startYear",
-        placeholder="year ({{ counts.travel_year }} entries)",
+        placeholder="year",
```

Comment:
Since you removed "year ({{ counts.travel_year }} entries", where exactly do the counts show up?

Reply:
For right now, counts for dates do not show up in the placeholder bar. We can hold off on this merge until I double-check with @ceserani

Reply:
Oh, I see, I was misled because I read the title "Counts accurate on Search page" and didn't realize you were removing counts.

Why keep the counts for the other fields though? Perhaps we could remove _all_ counts?

Reply:
Do you have a preference, @ceserani ?